assembl/static2/js/app/components/debate/thread/editPostForm.jsx

```diff
@@ -107,6 +108,7 @@ class EditPostForm extends React.PureComponent<void, EditPostFormProps, EditPost
               value={this.state.subject}
               handleTxtChange={this.updateSubject}
               maxLength={TEXT_INPUT_MAX_LENGTH}
+              isTopPost={this.props.isTopPost}
```

Comment:
please use a generic name, like readOnly

Reply:
+1 please don't use specific stuff in common components